events/importer/kulke.py

```diff
@@ -179,6 +180,40 @@ def get_event_name(event):
             return names[0]
 
 
+def parse_age_range(secondary_headline):
+    if not isinstance(secondary_headline, str):
+        return (None, None)
+
+    pattern = r'^\D*(\d{1,2}).(\d{1,2}).[v.|år].+$'
+    match = re.match(pattern, secondary_headline)
```

Comment:
This regexp does not probably do what you would like it to do.
```
>>> my_re = re.compile(r'^\D*(\d{1,2}).(\d{1,2}).[v.|år].+$')
>>> my_re.match('Yes 45678-åå').groups()
('45', '78')
>>> my_re.match('"Introduction 101"|by me').groups()
('1', '1')
```
The regular expression you're aiming at is probably:
`^\D*(\d\d?)-(\d\d?)[- ](v|år).*$`